chart/compass/templates/tests/ord-service/ord-service-test.yaml

```diff
@@ -176,6 +176,12 @@ spec:
               value: {{ .Values.global.tests.ordService.consumerTenantID }}
             - name: SKIP_TESTS_REGEX
               value: {{ .Values.global.tests.ordService.skipPattern }}
+            - name: SELF_REG_DISTINGUISH_LABEL_KEY
+              value: {{ .Values.global.tenantFetcher.server.subscriptionProviderLabelKey }}
+            - name: SELF_REG_DISTINGUISH_LABEL_VALUE
+              value: {{ .Values.global.tests.subscriptionProviderIdValue | quote }}
+            - name: SELF_REG_REGION
+              value: {{ .Values.global.tests.selfRegistration.region }}
```

Comment:
Why do we need these two new variables here? We already have the same values in the SUBSCRIPTION_CONFIG_PROVIDER_LABEL_KEY and SUBSCRIPTION_CONFIG_PROVIDER_ID ?